### PR TITLE
Display Members data on Members Page

### LINKED ISSUE #16: Display Members data on Members Page
Will need to pull in the members data
- each member should have an image, name, location, hobbies, links

### PR DESCRIPTION
**Pull request by galiotoweb**
## Description

Fetches the members data from the WWC Cincy database, maps through the object and displays each member's name, location, favorite techs, hobbies, and social links.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation content changes
- [ ] Other (please describe)

## List any open issues this change will close
#16 
## Describe how to test/validate these changes
Build repo locally 
## Screenshots (if appropriate)
Screenshot 2020-10-24 members page

### CODE REVIEW

File: src/pages/members.js [hunk 1]
```diff
@@ -1,12 +1,66 @@
...
+      }
+    } catch {
+      this.setState({ hasError: true });
+    }
+  }
+
+  // Backup Image in case url from database is broken
+  addDefaultSrc(ev) {
+    ev.target.src = "https://avatars3.githubusercontent.com/u/13358086?s=200&v=4"
+  }
```
Comment (alexxmitchell) [inline src/pages/members.js:28]: Nice!

File: src/pages/members.js [hunk 2]
```diff
@@ -1,12 +1,66 @@
...
+        </ul>
+      </div>}
+      <hr />
+    </div>)
+
+
+    return (<Layout>
+      <SEO title="Chapter Members" />
+      <h1>Chapter Members</h1>
+      {hasError && <p>Unable to fetch quotes</p>}
```
Comment (alexxmitchell) [inline src/pages/members.js]: Switch to `Unable to fetch members`

**Review by alexxmitchell — CHANGES_REQUESTED** [review]
Overall this looks good. Great idea to include a backup image. Just need to change the error message

File: src/pages/members.js (hunk 1 above)
Comment (galiotoweb) [reply src/pages/members.js:28]: Thanks! That was fun to figure out how to do

File: src/pages/members.js (hunk 2 above)
Comment (galiotoweb) [reply src/pages/members.js]: Good catch!

**Review by alexxmitchell — APPROVED** [review]

**Comment by alexxmitchell** [comment]
Closes #16

**Pull request merged**